GenerateAspNetCoreClient.Command/ClientModelBuilder.cs

```diff
@@ -177,12 +178,36 @@ private List<Parameter> GetParameters(ApiDescription apiDescription)
                     continue;
                 }
 
+                // IFormFile[], List<IFormFile>
+                if (paramType != null)
+                {
+                    bool isFormFileArray = paramType == typeof(IFormFile[]);
```

Comment:
Does it have to be specifically array or list? 
Does it make sense to check for IEnumerable<IFormFile> ?

Reply:
IEnumerable makes more sense and captures more outer collection types